ddl/column.go

```diff
@@ -165,11 +165,11 @@ func onAddColumn(d *ddlCtx, t *meta.Meta, job *model.Job) (ver int64, err error)
 		return ver, nil
 	}
 
-	failpoint.Inject("errorBeforeDecodeArgs", func(val failpoint.Value) {
+	if val, _err_ := failpoint.Eval(_curpkg_("errorBeforeDecodeArgs")); _err_ == nil {
```

Comment:
need `make failpoint-disable` - -

Reply:
Sorry, I run `make failpoint-enable` and commit and forgot to reset those changes.